Gemfile

```diff
@@ -39,6 +39,7 @@ gem "delayed_job_active_record", "~> 4.1" # ActiveRecord backend for Delayed::Jo
 gem "delayed_job_web", "~> 1.4" # Web interface for delayed_job
 gem "devise", "~> 4.6" # Flexible authentication solution for Rails
 gem "draper", "~> 3.1" # Draper adds an object-oriented layer of presentation logic to your Rails apps
+gem "dry-initializer", "~> 3.0" # Simple mixin of class methods :params and :options for instances
```

Comment:
I thought we agreed on removing this gem and using plain Ruby, maybe you force pushed something losing the changes?

Reply:
yes. I've made mess with force push. Sorry. Will try to be more careful in future...